### PR TITLE
docs: update changelog and release 0.2.0

### PR DESCRIPTION
**Pull request by ad1992**
pushed [here](https://codesandbox.io/s/excalidraw-v02-oc6un?file=/src/App.js)
- [x] update changelog
- [x] test
- [x] Add about extra API's exported in readme

### CODE REVIEW

File: src/packages/excalidraw/CHANGELOG.md [hunk 1]
```diff
@@ -12,15 +12,21 @@ The change should be grouped under one of the below section and must contain PR
...
 
 ### Features
 
+- feat: Add toast [#2772](https://github.com/excalidraw/excalidraw/pull/2772)
 - Add `cmd+o` shortcut to load scene [#2732](https://github.com/excalidraw/excalidraw/pull/2732)
 - Remove language picker, and add `langCode`, `renderFooter` [#2644](https://github.com/excalidraw/excalidraw/pull/2644):
   - BREAKING: removed the language picker from UI. It is now the host app's responsibility to implement a language picker if desirable, using the newly added [`renderFooter`](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#renderFooter) prop. The reasoning is that the i18n should be controlled by the app itself, not by the nested Excalidraw component.
   - Added [`langCode`](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#langCode) prop to control the UI language.
 - Add support for `exportToBackend` prop to allow host apps to implement shareable links [#2612](https://github.com/excalidraw/excalidraw/pull/2612/files)
+- Require use of a preset dialog size; adjust dialog sizing [#2684](<(https://github.com/excalidraw/excalidraw/pull/2684)>)
```
Comment (ad1992) [inline src/packages/excalidraw/CHANGELOG.md:25]:
```suggestion
- Require use of a preset dialog size; adjust dialog sizing [#2684](https://github.com/excalidraw/excalidraw/pull/2684)
```

File: src/packages/excalidraw/CHANGELOG.md [hunk 2]
```diff
@@ -12,15 +12,21 @@ The change should be grouped under one of the below section and must contain PR
 Please add the latest change on the top under the correct section.
 -->
 
-## [Unreleased]
+## 0.2.0
 
 ### Features
 
+- feat: Add toast [#2772](https://github.com/excalidraw/excalidraw/pull/2772)
```
Comment (ad1992) [inline src/packages/excalidraw/CHANGELOG.md:19]:
```suggestion
- Add toast [#2772](https://github.com/excalidraw/excalidraw/pull/2772)
```

File: src/packages/excalidraw/CHANGELOG.md [hunk 3]
```diff
@@ -31,7 +37,12 @@ Please add the latest change on the top under the correct section.
 
 ### Fixes
 
+- Fix typo for initialData and point all links to master [#2707](https://github.com/excalidraw/excalidraw/pull/2707)
+- Fix compile error [#2685](https://github.com/excalidraw/excalidraw/pull/2685)
+- Center zoom on iPhone and iPad [#2642](https://github.com/excalidraw/excalidraw/pull/2642)
 - Allow text-selecting in dialogs & reset cursor [#2783](https://github.com/excalidraw/excalidraw/pull/2783)
+- don't render due to zoom after unmount [#2779](https://github.com/excalidraw/excalidraw/pull/2779)
```
Comment (ad1992) [inline src/packages/excalidraw/CHANGELOG.md:44]:
```suggestion
- Don't render due to zoom after unmount [#2779](https://github.com/excalidraw/excalidraw/pull/2779)
```

File: src/packages/excalidraw/CHANGELOG.md [hunk 4]
```diff
@@ -46,7 +57,13 @@ Please add the latest change on the top under the correct section.
 
 ### Improvements
 
+- Perform lossless compression on all PNG images [#2740](https://github.com/excalidraw/excalidraw/pull/2740)
```
Comment (dwelle) [inline src/packages/excalidraw/CHANGELOG.md:60]: IMO this isn't related to the npm package. We don't publish those images.

File: src/packages/excalidraw/CHANGELOG.md [hunk 5]
```diff
@@ -47,6 +58,11 @@ Please add the latest change on the top under the correct section.
 ### Improvements
 
 - Added Zen Mode to the context menu [#2734](https://github.com/excalidraw/excalidraw/pull/2734)
+- Do not reset to selection for draw tool [#2721]((https://github.com/excalidraw/excalidraw/pull/2721)
+- Make dialogs look more like dialogs [#2686](<(https://github.com/excalidraw/excalidraw/pull/2686)>)
```
Comment (ad1992) [inline src/packages/excalidraw/CHANGELOG.md:62]:
```suggestion
- Make dialogs look more like dialogs [#2686](https://github.com/excalidraw/excalidraw/pull/2686)
```

File: src/packages/excalidraw/CHANGELOG.md [hunk 6]
```diff
@@ -47,6 +58,11 @@ Please add the latest change on the top under the correct section.
...
+- Browse libraries styles fixed [#2694](<(https://github.com/excalidraw/excalidraw/pull/2694)>)
```
Comment (ad1992) [inline src/packages/excalidraw/CHANGELOG.md:63]:
```suggestion
- Browse libraries styles fixed [#2694](https://github.com/excalidraw/excalidraw/pull/2694)
```

**Review by dwelle — APPROVED** [review]

**Comment by ad1992** [comment]
[v0.2.0](https://www.npmjs.com/package/@excalidraw/excalidraw) has been released 🚀

**Pull request merged**